test/integration/compliance_history_test.go

```diff
@@ -778,7 +778,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 		Expect(err).ToNot(HaveOccurred())
 
 		By("The Policy should be Compliant")
-		clusters = verifyPolicyOnAllClusters(ctx, policyNS, policyName, "Compliant", defaultTimeoutSeconds)
+		clusters = verifyPolicyOnAllClusters(ctx, policyNS, policyName, "Compliant", defaultTimeoutSeconds*4)
```

Comment:
I'd propose doubling the time first before quadrupling it since we don't want to extend the Integration tests longer than necessary (unless you know that it needs to be quadrupled?).